docs/contents/components/forms/segmented-control/accessibility.ja.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: SegmentedControl
+description: "`SegmentedControl`は、ユーザーが複数の選択肢の中から1つを選択するための使用されるコンポーネントです。"
+package_name: "@yamada-ui/segmented-control"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`SegmentedControlButton`にアイコンのみを使用する場合は、`SegmentedControlButton`に`aria-label`を設定します。
+
+```tsx live=false
+<Wrap gap="md">
+  <SegmentedControl>
+    <SegmentedControlButton aria-label="plus" value="plus">
+      <Plus />
+    </SegmentedControlButton>
+    <SegmentedControlButton aria-label="minus" value="minus">
+      <Minus />
+    </SegmentedControlButton>
+  </SegmentedControl>
+</Wrap>
+```
+
+## キーボード操作
+
+| キー         | 説明                                 | 状態 |
+| ------------ | ------------------------------------ | ---- |
+| `Tab`        | 選択されたボタンをフォーカスします。 | -    |
+| `ArrowLeft`  | 無効ではない前のボタンを選択します。 | -    |
+| `ArrowRight` | 無効ではない次のボタンを選択します。 | -    |
+
+## ARIAロールと属性
```

Comment:
Please add a description of `role="radiogroup"` in `SegmentedControl`.

Reply:
@hirotomoyamada 
There is no `radiogroup` role.

Reply:
Fixed.